plugins/plugin-docker/che-plugin-docker-client/src/main/java/org/eclipse/che/plugin/docker/client/params/PullParams.java

```diff
@@ -25,9 +26,11 @@
  */
 public class PullParams {
 
-    private String image;
-    private String tag;
-    private String registry;
+    private String      image;
+    private String      tag;
+    private String      registry;
+    private String      namespace;
```

Comment:
So, in case of docker.io it is mandatory, in case of another registry - optional. I removed the term `namespace`, but it content is included in full repo name